lib/src/components/NxAccordion/NxAccordion.scss

```diff
@@ -81,4 +81,14 @@
   border-top: 1px solid #e5e8f4;
   margin: 0 var(--nx-spacing-6x);
   padding: var(--nx-spacing-6x) 0;
+
+  .nx-list:first-child {
+    margin-top: calc(var(--nx-spacing-6x) * -1);
+
+    .nx-list__item:first-child {
+      &, > .nx-list__link, & > .nx-list__btn {
```

Comment:
Do we need to specify these selectors?

Reply:
> Do we need to specify these selectors?

It looks like we need it, without these selectors, the accordion divider line looks darker and thicker when the items are `NxList.LinkItem` or `NxList.ButtonItem`.

Reply:
> It looks like we need it, without these selectors, the accordion divider line looks darker and thicker when the items are NxList.LinkItem or NxList.ButtonItem.

We should have gallery examples of these cases then so they can have visual tests.  I'd say just put all three lists (normal, button, and link) in the same example as three separate accordions in a row